components/Footer/Footer.tsx

```diff
@@ -55,128 +60,88 @@ const Footer = ({
 		scrollToRef(ref)
 	}
 
+
 	return (
-		<div className="footer">
-			{/* Footer logo */}
-			<div className="footer-logo">
-				{/* SVG Logo */}
-				<svg width="75" height="75" viewBox="0 0 446 445" fill="none"
+		<div className="bg-[#f2f7ff] grid grid-cols-3 lg:grid-cols-5 gap-4 px-6 lg:px-[150px] py-6 items-center">
```

Comment:
Can we please use the tailwind configured `bg-zircon-50` for this?
![image](https://github.com/StudyCrew/StudyCrew/assets/74150974/6a119fd6-3426-4787-8e76-96fc86208af2)